velox/docs/configs.rst

```diff
@@ -354,3 +354,15 @@ Spark-specific Configuration
      - bool
      - true
      - If false, ``size`` function returns null for null input.
+   * - spark.runtime.bloom_filter.expected_num_items
+     - integer
+     - 1000000
+     - The default number of expected items for the runtime bloom filter in ``bloom_filter_agg`` function.
+   * - spark.runtime.bloom_filter.num_bits
+     - integer
+     - 8388608
+     - The default number of bits to use for the runtime bloom filter in ``bloom_filter_agg`` function.
+   * - spark.runtime.bloom_filter.max_num_bits
+     - integer
+     - 4194304
+     - The maximum number of bits to use for the runtime bloom filter in ``bloom_filter_agg`` function.
```

Comment:
Are these any limits to how low or high users can set this value? I believe 4194304 is the max. If so, would be nice to clarify.

Reply:
There is no limit in Spark, 4194304 is the default value, if it is too big, may raise OOM when creating bloom filter.

Reply:
Yes, there are no limits in Spark, but I believe Velox implementation has a limit, no?